cmd/wallet-web/src/config/credentialDisplayData.js

```diff
@@ -182,6 +182,26 @@ export default {
       fallback: 'Travel Card',
     },
   },
+  BookingReferenceCredential: {
+    id: ['$.id'],
+    brandColor: 'cobalt',
+    issuanceDate: ['$.issuanceDate'],
+    properties: {
+      name: {
+        path: ['$.credentialSubject.issuedBy'],
+        label: 'Issued By',
+      },
+      sex: {
```

Comment:
reference number is sex?

Reply:
My bad. Will update this.

Reply:
[Updated](https://github.com/trustbloc/wallet/compare/a4909217de1686fa97f8433b9fe69e3a74efa32f..9a6a6b3aece22d8cfa75cbddaa09985b8faaa1f1)